cms/src/components/groups/EditAclByGroup.vue

```diff
@@ -51,7 +51,7 @@ const duplicateGroup = (targetGroup: GroupDto) => {
                 <div class="flex items-center justify-between">
                     <div></div>
                     <div class="py-1">
-                        {{ assignedGroup.name }}
+                        {{ disabled ? assignedGroup._id : assignedGroup.name }}
```

Comment:
I do not think this is correct. Groups may be disabled for editing while the group doc is available (meaning we have the group name)